src/components/FullscreenHearing.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import {intlShape} from 'react-intl';
+import FullscreenPlugin from './FullscreenPlugin';
+import find from 'lodash/find';
+import {getHearingURL} from '../utils/hearing';
+import {userCanComment, userCanVote} from '../utils/section';
+import {Hearing, wrapHearingComponent} from './Hearing';
+
+
+class FullscreenHearing extends Hearing {
+
+  render() {
+    const hearing = this.props.hearing;
+    const user = this.props.user;
+
+    const hearingAllowsComments = !hearing.closed && (new Date() < new Date(hearing.close_at));
```

Comment:
It might seem a little overkill but I would say that this could be a function in the Hearing. Mainly to avoid having several places to change if the logic changes. This basically applies to the following as well `mainSection`, `mainSectionCommentable` and possibly even `mainSectionVotable`.

In my opinion, since this class is only to provide `fullscreen`, it should not be possible that at some point, its behavior differs from the regular hearing. So, if possible, it is nicer to push all the logic to the Hearing class.